### PR TITLE
Add js-beautify.vm

### LINKED ISSUE #1075: npm package proposal: js-beautify.vm
### Package Name

js-beautify

### Tool Name

js-beautify

### Package type

NODE

### Version

1.15.1.20240605

### Category

Javascript

### Tool's authors

Einar Lielmanis

### Tool's description

Beautifier for JavaScript

### Dependencies

_No response_

### Why is this tool a good addition?

While CyberChef includes a JavaScript beautifier, it fails on large files. Ideally `js-beautify` could be used in those situations.

### Extra information

_No response_

### PR DESCRIPTION
**Pull request by d35ha**
Closes #1075.

### CODE REVIEW

File: packages/js-beautify.vm/js-beautify.vm.nuspec [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>js-beautify.vm</id>
+    <version>1.15.1.20240930</version>
+    <authors>beautifier.io</authors>
+    <description>This little beautifier will reformat and re-indent bookmarklets, ugly JavaScript, unpack scripts packed by Dean Edward's popular packer, as well as partly deobfuscate scripts processed by the npm package javascript-obfuscator.</description>
```
Comment (Ana06) [inline packages/js-beautify.vm/js-beautify.vm.nuspec:7]:
```suggestion
    <description>JavaScript beautifier and deobfuscator.</description>
```
According to [our documentation](https://github.com/mandiant/VM-Packages/wiki/Chocolatey-packages#description):
> The description should be short and not include version specific details or other information that is likely to change in a future version.

**Review by Ana06 — APPROVED** [review]
Tested and it works nicely! Just one improvement suggestion :wink:

File: packages/js-beautify.vm/js-beautify.vm.nuspec (hunk 1 above)
Comment (d35ha) [reply packages/js-beautify.vm/js-beautify.vm.nuspec]: Done.

**Review by Ana06 — APPROVED** [review]

**Comment by Ana06** [comment]
Thanks @d35ha! :tada:

**Pull request merged**